dev-support/docker/pkglist

```diff
@@ -36,5 +36,4 @@ python3.6
 python3.7
 python3.8
 tox
-virtualenv
```

Comment:
nit: since this will affect the dev docker container as well please add virtualenv to the pip install line there: https://github.com/apache/beam/blob/dd15d59fc82fb50d3cd511370bd78034c6a99074/dev-support/docker/Dockerfile#L90

Reply:
oh good catch!
Dependency added.
Thanks for the heads up!